### PR TITLE
[Flake8] Add more description about `plugins` option

### PR DESCRIPTION
**Pull request by ybiquitous**
About a version specifier or VCS URL format. See <https://pip.pypa.io/en/stable/reference/pip_install/>.

### CODE REVIEW

File: docs/tools/python/flake8.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ linter:
     plugins:
       - flake8-bandit
       - flake8-builtins==1.4.1
-      - flake8-mypy>=17.3.3
```
Comment (ybiquitous) [inline docs/tools/python/flake8.md:39]: [note] `flake8-mypy` is dead: https://github.com/ambv/flake8-mypy

**Review by yykamei — APPROVED** [review]
LGTM

**Comment by ybiquitous** [comment]
@gnakagaw ping?

**Review by gnakagaw — APPROVED** [review]
LGTM

I'm sorry for my late reaction.

**Pull request merged**